subxt/src/error/mod.rs

```diff
@@ -162,3 +162,14 @@ pub enum StorageAddressError {
         fields: usize,
     },
 }
+
+/// This trait is implemented on the statically generated root ModuleError type
+#[doc(hidden)]
+pub trait RootError: Sized {
+    /// Given details of the pallet error we want to decode
+    fn root_error(
+        pallet_bytes: &[u8; 4],
```

Comment:
Nit: Not a big deal at all but I'd either go for `[u8; 4]` (no need for reference since it's only 4 bytes) or `&[u8]` (gives a little room for growth if the error type grows from `[u8;4]` for some reason) :)